src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -22,7 +22,7 @@
 
 """Module to control interaction with MAPDL through Python"""
 
-import atexit
+# import atexit
```

Comment:
All the changes have been commented, is there a reason to keep them?

Reply:
because I am still checking their impact... I might remove them, and create an issue to remind me to check this.